src/biotite/database/uniprot/query.py

```diff
@@ -11,7 +11,7 @@
 from .check import assert_valid_response
 
 
-_base_url = "https://www.uniprot.org/uniprot/"
+_base_url = "https://www.uniprot.org/uniprotkb/"
```

Comment:
The new search URL is slightly different.

```suggestion
_base_url = "https://rest.uniprot.org/uniprotkb/search"
```